cryo/dataset.py

```diff
@@ -0,0 +1,172 @@
+"""Open datasets and process them to be used by a neural network."""
+
+import json
+import os
+import numpy as np
+import h5py
+import torch
+from torch.utils.data import DataLoader, random_split
+import functools
+from PIL import Image
+
+CUDA = torch.cuda.is_available()
+
+KWARGS = {'num_workers': 1, 'pin_memory': True} if CUDA else {}
+
+
+def open_dataset(path, new_size, is_3d):
+    """Open datasets and processes data in ordor to make tensors.
+
+    Parameters
+    ----------
+    path : string,
+        Path.
+    new_size : int.
+    is_3d : boolean
+        If 2d or 3d.
+
+    Returns
+    -------
+    tensor
```

Comment:
If I understand the code correctly, this tensor is specifically a `torch` tensor. Would it be readable by other backends such as `tensorflow` or `numpy`? If not, please mention it here.